src/lens.jl

```diff
@@ -1,13 +1,16 @@
 export Lens, set, get, modify
 export @lens
 export set, get, modify
+using ConstructionBase
 export setproperties
+export constructorof
+
 
 import Base: get
 using Base: setindex, getproperty
 
 """
-    Lens
+	Lens
```

Comment:
Can you please use four spaces for indentation? Looks like you are using two tabs?

Reply:
Definitely! Unsure how that happened as I usually use 4 spaces.